.gitlab-ci.yml

```diff
@@ -6,16 +6,31 @@ include:
   - project: hpc/gitlab-upload-logs
     file: enable-upload.yml
 
+variables:
+  CVF_BRANCH:
+    description: Branch of the channel validation framework (CVF) to trigger the CI of
+    value: main
+  SPACK_BRANCH:
+    description: Branch of BlueBrain Spack to use for the CI pipeline
+    value: develop
+  SPACK_DEPLOYMENT_SUFFIX:
+    description: Extra path component used when finding deployed software. Set to something like `pulls/1497` use software built for https://github.com/BlueBrain/spack/pull/1497. You probably want to set SPACK_BRANCH to the branch used in the relevant PR if you set this.
```

Comment:
This is just a boilerplate thing that I've copied into most of our GitLab CI YAML files because it means the page for manually launching pipelines:
<img width="1290" alt="Screenshot 2022-05-11 at 17 31 19" src="https://user-images.githubusercontent.com/6459623/167889065-b90c6840-2ef7-4aa0-8a66-f9c00c70a2e4.png">
is pre-populated with helpful fields.

If you want to test NMODL against a BlueBrain/spack PR then you can set this so that **as well as** checking out the feature branch of Spack in the NMODL CI, you use software from the GPFS prefix that the Spack CI uses for that PR. This means that if, for example, you want to run the NMODL CI with a new `nvc++` version then that just works, with all of the hard work of installing and configuring `nvhpc` done by the Spack pipeline before you launch the NMODL one. Similarly if you want to test an update to the Spack recipe of some library that appears in the dependency tree of NMODL, you will pick up a version of it built by the Spack pipeline.